scripts/repl.js

```diff
@@ -109,6 +109,10 @@
     var state = UriUtils.parseQuery();
     this.options = _.assign(new Options(), state);
 
+    if (window.localStorage) {
```

Comment:
You may want to move the local storage call up above where options are set on the repl instance (so that options can be persisted too).